terraform/hcl/applicable/req_sw_terraform_aws_cloudtrail_encrypt.tf

```diff
@@ -0,0 +1,5 @@
+resource "aws_cloudtrail" "vulnerable_example" {
+  s3_bucket_name = "dummy"
+  name = "dummy"
+  # kms_key_id is not set
+}
```

Comment:
<div align="center"> 

### 🛠️ Infrastructure as Code Vulnerability
	
| Severity | Finding |
| :---: | :---: |
| ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableMediumSeverity.png)<br> | kms_key_id='' was detected |

</div>

<details>
<summary> <b>Full description</b> </summary>
<br>

Resource `aws_cloudtrail` should have key `kms_key_id`

Vulnerable example -
```
resource "aws_cloudtrail" "vulnerable_example" {
  # kms_key_id is not set
}
```

Secure example -
```
resource "aws_cloudtrail" "secure_example" {
  kms_key_id = aws_kms_key.example.arn
}
```

</details>


---
<div align="center">

[🐸 JFrog Frogbot](https://github.com/jfrog/frogbot#readme)

</div>